packages/manager/modules/support/src/tickets/new-ticket/new-ticket.html

```diff
@@ -93,10 +93,16 @@
     <p>
         <strong data-translate="ovhManagerSupport_new_creation_error"></strong>
     </p>
-    <p>
-        <span data-ng-bind="$ctrl.error.message"></span>
-        <br />
-        <small data-ng-bind="$ctrl.error.queryId"></small>
+    <p data-ng-switch="$ctrl.error.class">
+        <span
+            data-ng-switch-when="Client::BadRequest::AlreadyOpened"
```

Comment:
Not a big fan to do a switch directly in html based on API results.

Reply:
Why ?

Reply:
Because the way we fetch and manage the data/errors are not the html's responsibility.